### ISSUE TITLE
cometvisu-client.js: write function lacks sending session to backend

### ISSUE THREAD

**Issue by tuxedo0801** [CometVisu/CometVisu#199]
login function gets the session. Read function sends the session with every read.
But writing is totally done without session:
```
  /**
   * This function sends a value
   * @method write
   */
  this.write = function( address, value )
  {
    /**
     * ts is a quirk to fix wrong caching on some Android-tablets/Webkit;
     * could maybe selective based on UserAgent but isn't that costly on writes
     */
    var ts = new Date().getTime();
    $.ajax({
      url:      this.urlPrefix + 'w',
      dataType: 'json',
      context:  this,
      data:     'a=' + address + '&v=' + value + '&ts=' + ts
    });
  }
```

**Comment by tuxedo0801** [comment]
see PR #209

**Comment by ChristianMayer** [comment]
PR #209 merged with commit 8da97af
